NEWS.md

```diff
@@ -44,15 +44,11 @@
 
 6. v1.12.0 did not compile on Solaris 10 using Oracle Developer Studio 12.6, [#3285](https://github.com/Rdatatable/data.table/issues/3285). Many thanks to Prof Ripley for providing and testing a patch. For future reference and other package developers, a `const` variable should not be passed to OpenMP's `num_threads()` directive otherwise `left operand must be modifiable lvalue` occurs.
 
-7. `getDTthreads()` respected the `OMP_NUM_THREADS` environment variable but not `OMP_THREAD_LIMIT`, [#3300](https://github.com/Rdatatable/data.table/issues/3300). There are two very similar OpenMP functions: `omp_get_max_threads()` and `omp_get_thread_limit()`. It now calls both and chooses the minimum. Note that these environment variables should be set before the R session starts. Using the R command `Sys.setenv()` to set them is too late because the OpenMP runtime is already running by then; use `setDTthreads()` instead.
+7. `getDTthreads()` respected the `OMP_NUM_THREADS` environment variable but not `OMP_THREAD_LIMIT`, [#3300](https://github.com/Rdatatable/data.table/issues/3300). Note that these environment variables should be set before the R session starts; using the R command `Sys.setenv()` to set them is too late because the OpenMP runtime is already running by then (use `setDTthreads()` instead). Further, the default number of threads is reduced to half the number of logical CPUs. The previous default of using all logical CPUs was reported to cause significant slow downs when other non-trivial processes are also running, [#3395](https://github.com/Rdatatable/data.table/issues/3395), [#3298](https://github.com/Rdatatable/data.table/issues/3298). The number of logical CPUs is determined by `omp_get_num_procs()`. This value has been added to the output of `getDTthreads(verbose=TRUE)`.
```

Comment:
We need more verbose message that all resources might not be utilised, ideally on top of NEWS. AFAIU there are much more users having no problems using all logical cores. Maybe message even in package startup message? Something like
> data.table is using X cores while Y logical cores are available, for details see `?setDTthreads`.